code/modules/projectiles/guns/breacher.dm

```diff
@@ -245,7 +245,7 @@
 		var/mob/living/carbon/human/H = src
 		H.take_overall_damage(35, 10)
 	else
-		ex_act(2)
+		explosion_act(600, null)
```

Comment:
Should deal roughly 100-150 damage to mobs instead like it used to (ex_act(2) did 120 damage, ~60 to roaches), otherwise this allows instantly gibbing antags like blitzshells, or very very quickly removing kaisers.